assets/js/api/CustomTranslationApi.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery'
+
+export class CustomTranslationApi {
+  constructor (programSection) {
+    this.programSection = programSection
+  }
+
+  getCustomTransaltion (programId, language, successCallback, errorCallback) {
+    $.ajax({
+      url: '../translate/custom/project/' + programId + '?field=' + this.programSection + '&language=' + language,
+      type: 'get',
+      success: function (data) {
+        successCallback(data)
+      },
+      error: function () {
+        errorCallback()
+      }
+    })
+  }
+
+  deleteCustomTranslation (programId, language, successCallback, errorCallback) {
+    $.ajax({
+      url: '../translate/custom/project/' + programId + '?field=' + this.programSection + '&language=' + language,
+      type: 'delete',
+      success: function () {
```

Comment:
i think you can do `success: successCallback` directly, as in `successCallback` is type function

Reply:
Added the callback directly here. Although for functions that have a parameter passed back this doesn't seem to work

Reply:
```
      success: successCallback,
      error: errorCallback
```

Reply:
Updated, I was trying successCallback(data) and data was undefined. This works though 👍